spec/factories/news_items.rb

```diff
@@ -1,6 +1,8 @@
 FactoryGirl.define do
   factory :news_item do
-    title "my-title"
+    title { Faker::Lorem.sentence }
+    state 'draft'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.